providers/src/airflow/providers/google/cloud/operators/dataproc.py

```diff
@@ -1962,6 +1962,9 @@ def __init__(
         polling_interval_seconds: int = 10,
         cancel_on_kill: bool = True,
         wait_timeout: int | None = None,
+        ol_inject_parent_job_info: bool = conf.getboolean(
```

Comment:
If we're already using rather long option names, we can expand the `ol` abbreviation